lib/moirai/pull_request_creator.rb

```diff
@@ -1,41 +1,45 @@
 class Moirai::PullRequestCreator
-  BRANCH_NAME = "moirai-translations"
-
   def self.available?
-    defined?(Octokit)
+    !!defined?(Octokit)
   end
 
+  attr_reader :github_repo_name, :github_access_token, :github_client, :github_repository, :branch_name
+
   def initialize
     @github_repo_name = ENV["MOIRAI_GITHUB_REPO_NAME"] || Rails.application.credentials.dig(:moirai, :github_repo_name)
     @github_access_token = ENV["MOIRAI_GITHUB_ACCESS_TOKEN"] || Rails.application.credentials.dig(:moirai, :github_access_token)
-    @client = Octokit::Client.new(
-      access_token: @github_access_token
-    )
+    @github_client = Octokit::Client.new(access_token: github_access_token)
+    @github_repository = github_client.repo(github_repo_name)
   end
 
   def create_pull_request(translations_array)
-    repo = @client.repo(@github_repo_name)
-    default_branch = repo.default_branch
+    @branch_name = "moirai-translations-#{Time.current.strftime("%F-%H-%M-%S")}"
```

Comment:
branch name must be different each time, otherwise there might be conflicts, for example when we have different CI, running in parallel. Is also correct that a new branch is created each time.